dbcrossbarlib/src/drivers/postgres_shared/create_table_sql.rustpeg

```diff
@@ -10,25 +10,24 @@
 //!
 //! [peg]: https://github.com/kevinmehall/rust-peg
 
-use crate::schema::{Column, DataType, Table};
+use super::super::{PgColumn, PgCreateTable, PgDataType, PgScalarDataType, Srid};
 
 /// A `CREATE TABLE` expression.
-pub create_table -> Table
+pub create_table -> PgCreateTable
```

Comment:
it's funny that, in addition to everything else, you also have a format CREATE TABLE grammar in here

Reply:
Yup, this is all part of the master plan. :-) When we parse these, we translate them into the cross-database `Table` type. This means you can load a CSV file into a BigQuery table using a PostgreSQL `CREATE TABLE` statement.

Eventually, I want to have grammars for all the databases we support, so that you don't need to care whether you have `pg_table.sql` or `bq_table.json` or whatever.

Also, we can extract a schema directly from a CSV file—we just use `text` for all the column types. And then you can write that schema out as a PostgreSQL `CREATE TABLE`. This is going to save @wboykinm some time, I bet.